resip/recon/ConversationManager.hxx

```diff
@@ -483,6 +486,18 @@ public:
    virtual void onDtmfEvent(ParticipantHandle partHandle, int dtmf, int duration, bool up) = 0;
 
    ///////////////////////////////////////////////////////////////////////
+   // Pager Message Handlers /////////////////////////////////////////////
+   ///////////////////////////////////////////////////////////////////////
+
+   virtual void onMessageArrived(resip::ServerPagerMessageHandle handle, const resip::SipMessage& message) = 0;
```

Comment:
The method onMessageArrived() is called at [1] using the ServerPagerMessageHandler that was set at resip/recon/UserAgent.cxx:

mDum.setServerPagerMessageHandler(mConversationManager);

The method onMessage() is coming from InviteSessionHandler that ConversationManager derived from. Do you think I could use this same method for incoming text messages?

[1] https://github.com/resiprocate/resiprocate/blob/master/resip/dum/ServerPagerMessage.cxx#L86